### PR TITLE
HPCC-16768 BS, HR, SR Translations

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-16768

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq @emuharemagic Please Review

File: esp/src/eclwatch/nls/hr/hpcc.js [hunk 1]
```diff
@@ -88,8 +111,13 @@
     Debug: "Otklonite Neispravnosti",
     DEF: "DEF",
     Defaults: "Unaprijed Definisane Vrijednosti",
+    Definition: "Definicija",
     Delete: "Obrišite",
+    DeleteBinding: "Izbrišite Vezivanje",
     Deleted: "Obrisan",
+    DeletedBinding: "Izbrisano Vezivanje",
+    DeleteDirectories: "Eliminišite prazane direktirije. Želite li nastaviti?",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/hr/hpcc.js]: "prazne" instead of "prazane"

File: esp/src/eclwatch/nls/sr/hpcc.js [hunk 2]
```diff
@@ -29,22 +36,33 @@
     AllowFull: "<center>Дозволите<br>Потпун</center>",
     AllowRead: "<center>Дозволите<br>Читање</center>",
     AllowWrite: "<center>Дозволите<br>Писање</center>",
+    AllQueuedItemsCleared: "Сви послови из реда чекања су очишћени. Тренутно активни посао наставња са извршавањем.",
     ANY: "Било Који",
+    AnyAdditionalProcessesToFilter: "33/5000Има ли још процеса за филтрирање",
```
Comment (emuharemagic) [inline esp/src/eclwatch/nls/sr/hpcc.js]: 33/5000 part needs to be dropped

**Comment by GordonSmith** [comment]
@emuharemagic Changes Made.

**Review by emuharemagic — APPROVED** [review]

**Pull request merged**